File: sdklibrary_international/src/main/java/com/sdk/sdklibrary/config/SDKStatusCode.java
package com.sdk.sdklibrary.config;

/**
 * Created by tzw on 2018/6/5.
 * 全局状态码:
 */

public final class SDKStatusCode {
    public static final int SUCCESS = 0;
    public static final int FAILURE = 1;
    public static final int OTHER = 2;
    public static final int CANCEL = 3;

    public static final int PAY_SUCCESS = 200;
    public static final int PAY_FAILURE = 404;
    public static final int PAY_OTHER = 500;

    public static final int LOGOUT_SUCCESS = 801;
    public static final int LOGOUT_FAILURE = 802;
    public static final int LOGOUT_CANCEL = 803;
    public static final int LOGOUT_OTHER = 884;
}

File: sdklibrary_kt/src/main/java/com/sdk/sdklibrary/config/SDKStatusCode.kt
package com.sdk.sdklibrary.config

/**
 * @author colin
 * Date:2023-02-08
 * 全局状态码:
 */
object SDKStatusCode {
    const val SUCCESS = 0
    const val FAILURE = 1
    const val OTHER = 2
    const val CANCEL = 3
    const val PHONE_REGISTER_SUCCESS = 500
    const val PHONE_REGISTER_FAILURE = 501
    const val PHONE_REGISTER_OTHER = 503
    const val PHONE_REGISTER_CANCEL = 502
    const val PAY_SUCCESS = 200
    const val PAY_FAILURE = 201
    const val PAY_CANCEL = 202
    const val LOGOUT_SUCCESS = 801
    const val LOGOUT_FAILURE = 802
    const val LOGOUT_CANCEL = 803
    const val LOGOUT_OTHER = 884
}